code/modules/roguetown/roguejobs/alchemist/reagents.dm

```diff
@@ -19,11 +19,28 @@
 		//some peeps dislike the church, this allows an alternative thats not a doctor or sleep. 
 		M.heal_wounds(2) //at a motabalism of .5 U a tick this translates to 80WHP healing with 20 U Most wounds are unsewn 15-100. This is powerful on single wounds but rapidly weakens at multi wounds.
 		M.update_damage_overlays()
```

Comment:
> overdose_threshold = 0
Won't this mean potions have a overdose effect but no one can ever actually od on them?